public/js/SceneManager.js

```diff
@@ -239,32 +165,78 @@ class SceneManager2 {
     this.sceneEntities.push(new ParallaxAnimatedBackground(this.game, this,
       new Animation(this.game.assetManager.getAsset('../img/zerlin at fire.png'), 0, 0, 600, 600, .125, 6, true, false, .5),
       1, 800, 470, 450));
-    this.sceneEntities.push(new Overlay(this.game, true, OPENING_OVERLAY_TIME));
+    this.sceneEntities.push(new Overlay(this.game, true, smc.OPENING_OVERLAY_TIME));
 
     // start opening scene music
+
+    //link up html buttons to try to login or register a user
+    this.playGame = false; //for now use button to start the game
+    document.getElementById("loginButton").addEventListener('click', () => {
+      //do user validation here
+      // if (fields not blank) {
+      //   if (is user or can register) {
+      //     set playGame to true
+      //     hide login form
+      //   } else {
+      //     prompt bad info
+      //   }
+      // }
+
+      //user can't skip opening camera pan down
+      if (this.openingSceneTimer > smc.OPENING_SCENE_STOP_CAMERA_PAN) {
+        this.playGame = true;
+        document.getElementById("formOverlay").style.display = "none";
+      }
+    });
   }
 
   openingSceneUpdate() {
-    if(this.game.click) {
+    if (!this.canPause && this.game.keys['Enter']) {
       this.startLevelTransitionScene(); //for going strait into the lvl
+      document.getElementById("formOverlay").style.display = "none";
     }
+
+    this.musicMenu.update();
+
     this.openingSceneTimer += this.game.clockTick;
-    if (this.openingSceneTimer < OPENING_SCENE_STOP_CAMERA_PAN) {
-      this.camera.y = -Math.pow(this.openingSceneTimer - OPENING_SCENE_STOP_CAMERA_PAN, 2) * 280;
-    } else if (!this.seq1FadeOut && this.openingSceneTimer > OPENING_SCENE_FIRST_FADE_OUT_TIME) {
-      this.seq1FadeOut = true;
-      this.sceneEntities.push(new Overlay(this.game, false, OPENING_OVERLAY_TIME));
-    } else if (!this.seq2 && this.openingSceneTimer > OPENING_OVERLAY_TIME + OPENING_SCENE_FIRST_FADE_OUT_TIME) {
-      this.seq2 = true;
-      this.sceneEntities.push(new TextScreen(this.game, OPENING_MESSAGE));
-      this.sceneEntities.push(new Overlay(this.game, true, OPENING_OVERLAY_TIME));
-    } else if (!this.seq2FadeOut && this.openingSceneTimer > OPENING_OVERLAY_TIME + OPENING_SCENE_FIRST_FADE_OUT_TIME + OPENING_MESSAGE_TIME) {
-      this.seq2FadeOut = true;
-      this.sceneEntities.push(new Overlay(this.game, false, OPENING_OVERLAY_TIME));
-    } else if (this.openingSceneTimer > OPENING_OVERLAY_TIME + OPENING_SCENE_FIRST_FADE_OUT_TIME + OPENING_MESSAGE_TIME + OPENING_OVERLAY_TIME) {
-      this.startLevelScene();
+
+    //play the sweet rotating stars until the user clicks play or logs in
+    if (!this.playGame && this.openingSceneTimer < smc.OPENING_SCENE_STOP_CAMERA_PAN) {
```

Comment:
Whats the reason for moving the camera movement logic into a different if/else branch? Also, you mentioned a glitch with it skipping the "Here begins a new journey..." part? the only change we need to make it is for this.startLevelScene at the bottom to be this.startLevelTransitionScene. I must have just left that in for testing.

Reply:
I wanted to change the first statement so that the camera will pan down onto zerlin by the fire and will stay there with the fire flickering and the stars rotating until a user either logins/registers successfully or they hit the play button that I would like to add. This way we would have a cool login area

Reply:
where you could see have an interesting animation to look at until you start up the game

Reply:
Oh yes, thats a pretty cool thought. I was thinking of having an actual screen dedicated to the login so that this doesn't start until they enter, but I like this better. However, my intention was to start the main musical theme when this scene started. And I'm not sure that starting the music before they login is as dramatic as we want it, but we'll see when the scene is finished